### ISSUE TITLE
Theme 18.48-git-201705102210 - JavaScript seems broken.

### ISSUE THREAD

**Issue by joergma** [qooob/authentic-theme#763]
I have the latest theme (18.48) and Webmin (1.840) installed.
* My server-side OS and its version: Ubuntu Linux 16.04 (Linux 4.4.0-77-generic on x86_64)
* My client-side OS, browser and its versions: Windows 10 x64 1607, Firefox 53.0 portable (no Add-ons)
* What are the steps to reproduce the issue:

All pages called from the main menu are scrolling down automatically and instantly. Unable to scroll up. All JavaScript-driven buttons doesn't work. Opening a page in a new tab is possible and there is no automatic scrolling. However, the buttons still doesn't work. So I cannot change the theme or install an older Authentic Theme.

Same issues with Opera, Internet Explorer and Iron browser.
In other words: I cannot manage my server via browser. Any suggestions?

**Comment by qooob** [comment]
Hi,

I can not reproduce this issue. Try updating to the very latest commit, restart _Webmin/Usermin_, clear your browser's cache and try again.

The better solution is to make sure that the cache gets really cleared, update using `theme-update.sh` script in console, following [these instructions](https://github.com/qooob/authentic-theme#how-to-update-authentic-theme-manually).

**Comment by joergma** [comment]
Okay, I've done the following:

Updated using theme-update.sh & restarted webmin - same result.
Logged in from another real machine (Windows 7 x64) with several cache-cleared browsers - no luck.
Switched via shell to the default blue-theme (so called *Gray Framed Theme*) - this theme works flawlessly.
Deleted the Authentic Theme & rebooted the server.
Re-installed the Authentic Theme and switched to it - Authentic Theme doesn't work.
Switched again via shell to the default blue-theme.

Well, excluding the Authentic Theme there are two reasons left: Webmin or the last server OS update (about a week ago, but no Perl update, only Kernel). Should I solve this issue, I'll let you know.

**Comment by qooob** [comment]
My last most fancy guess:

`rm -rf /usr/share/webmin/authentic-theme`

Then

`reboot`

Then going to Webmin Configuration/Themes from Gray Theme, install Authentic Theme using upload or URL.

I remember very fancy, episodic cache, server-side issues happened with me as well. I could never reproduce it or link it to the theme.

**Comment by joergma** [comment]
Done - thanks for your suggestion. But now, Authentic Theme refuses completely to show up, despite switching to it, even after a cache cleaning and another reboot. The Gray Theme stays. For now, I give up and switch back to the Gray Theme. I like your Theme but maybe it doesn't like me ;)

BTW: I forgot to mention that my server is a KVM with enough RAM, no swap in use.
I'll wait some days and try again. Should I solve this issue, I'll let you know.
